### ISSUE TITLE
Warnings from unsafe methods

### ISSUE THREAD

**Issue by mskzo** [react-select-oss/react-select#18]
It's not a BUG but a warning. 

index.js:1 Warning: Using UNSAFE_componentWillReceiveProps in strict mode is not recommended and may indicate bugs in your code. See https://reactjs.org/link/unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://reactjs.org/link/derived-state

Please update the following components: Async, AutosizeInput, Select

**Comment by Methuselah96** [comment]
Thanks for the issue! It looks like https://github.com/react-select-oss/react-select/pull/17 resolves `Async`, but we still need to address `AutosizeInput` and `Select`. PRs are welcome!

**Comment by mskzo** [comment]
Hi,

How can i test the current dev version? Do you have any published package? 

BR

**Comment by Methuselah96** [comment]
[`react-select-oss@3.1.1`](https://github.com/react-select-oss/react-select/releases/tag/react-select-oss%403.1.1) has all of the latest changes from `master`.

**Comment by mskzo** [comment]
I tested it again and the warning is not solved. 
I removed the dependency and installed it again with npm remove and npm install.
BR

**Comment by Methuselah96** [comment]
Yes, sorry the issue has not been resolved yet. I was just noting that https://github.com/react-select-oss/react-select/pull/17 might fix part of the problem, but it still hasn't been merged or released yet. Fixing the complete issue has not been done yet.
